core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/commit/CommitInfoGetRequest.java

```diff
@@ -34,4 +38,9 @@ protected CommitInfoSearchRequestBuilder createSearchRequestBuilder() {
 		return new CommitInfoSearchRequestBuilder();
 	}
 
+	@Override
+	public Permission getPermission() {
+		return new Permission(Permission.BROWSE, Permission.ALL);
```

Comment:
Just adding a note here: the required permission might be dependent on the (repository) context the request is executed in. As an example, the user might gain access to the commit history of one repository, but be denied when trying to do the same for another.